runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.service.ts

```diff
@@ -438,4 +452,16 @@ export class CredentialService {
             }
         };
     }
+
+    private throwExceptionIfUserHasReadAcces(collection: CredentialCollection, user: User) {
```

Comment:
I'd rename it to something like `checkUserReadAccessOrThrow` because calling this function is a bit confusing